File: app/src/main/java/com/example/helloworld/ItalyChartScreen.java
package com.example.helloworld;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.anychart.anychart.AnyChart;
import com.anychart.anychart.AnyChartView;
import com.anychart.anychart.DataEntry;
import com.anychart.anychart.Pie;
import com.anychart.anychart.ValueDataEntry;

import java.util.ArrayList;
import java.util.List;

public class ItalyChartScreen extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_chart_screen);

        Pie pie = AnyChart.pie();

        List<DataEntry> italy = new ArrayList<>();
        italy.add(new ValueDataEntry("Zakażeni", 105792));
        italy.add(new ValueDataEntry("Zmarli", 12428));
        italy.add(new ValueDataEntry("Wyleczeni", 15729));

        pie.setData(italy);
        pie.setTitle("COVID-19");

        AnyChartView anyChartView = (AnyChartView) findViewById(R.id.any_chart_view);
        anyChartView.setChart(pie);
    }
}

File: app/src/main/java/com/example/helloworld/ItalyChartScreen.kt
package com.example.helloworld

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.anychart.anychart.AnyChart
import com.anychart.anychart.AnyChartView
import com.anychart.anychart.DataEntry
import com.anychart.anychart.ValueDataEntry
import java.util.*

class ItalyChartScreen : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_chart_screen)
        val pie = AnyChart.pie()
        val italy: MutableList<DataEntry> = ArrayList()
        italy.add(ValueDataEntry("Zakażeni", 105792))
        italy.add(ValueDataEntry("Zmarli", 12428))
        italy.add(ValueDataEntry("Wyleczeni", 15729))
        pie.setData(italy)
        pie.setTitle("COVID-19")
        val anyChartView = findViewById<View>(R.id.any_chart_view) as AnyChartView
        anyChartView.setChart(pie)
    }
}